pandas/core/indexing.py

```diff
@@ -113,38 +110,6 @@ def __call__(self, axis=None):
         new_self.axis = axis
         return new_self
 
-    def __iter__(self):
-        raise NotImplementedError("ix is not iterable")
-
-    def __getitem__(self, key):
```

Comment:
Could we keep this method? (or part of it, for compat with released geopandas)

Reply:
That makes sense.  Possibly with a warning telling users they need to upgrade geopandas?

Reply:
Yep, that sounds good. But maybe start having it as a DeprecationWarning? Then we can easily switch to FutureWarning once there is an actual geopandas release.